Localization/fr-FR/about.po

```diff
@@ -0,0 +1,36 @@
+msgid ""
+msgstr ""
+"FAH-Translate-Schema-Version: 1.0\n"
+"URL: https://foldingathome.org/about/\n"
+"Source-Language: en-US\n"
+"Source-Version: 1.0\n"
+"Source-Last-Modified: 20200705\n"
+"Target-Language: fr-FR\n"
+"Target-Version: 1.0\n"
+"Target-Last-Modified: 20200711\n"
+"Project-Id-Version: \n"
+"POT-Creation-Date: \n"
+"PO-Revision-Date: \n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Language: fr_FR\n"
+"X-Generator: Poedit 2.3.1\n"
+
+msgid "À propos"
+msgstr ""
```

Comment:
The idea is to keep the `msgid` the original and put the translation into the `msgstr`:
```suggestion
msgid "About"
msgstr "À propos"
```

Reply:
Just to point it out, this was just one example. :) Of course the same has to be done for all the other entries.

Reply:
Is it possible to review again ?